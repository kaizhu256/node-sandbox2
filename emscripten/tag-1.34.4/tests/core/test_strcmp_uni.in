#include <stdio.h>
#include <string.h>
int main() {
#define TEST(func)                                  \
  {                                                 \
    char *word = "WORD";                            \
    char wordEntry[2] = {-61, -126}; /* "Ã‚"; */    \
    int cmp = func(word, wordEntry, 2);             \
    printf("Compare value " #func " is %d\n", cmp); \
  }
  TEST(strncmp);
  TEST(strncasecmp);
  TEST(memcmp);
}
