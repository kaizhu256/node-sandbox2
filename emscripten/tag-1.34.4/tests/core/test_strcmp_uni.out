Compare value strncmp is -108
Compare value strncasecmp is -76
Compare value memcmp is -108
